### ISSUE TITLE
source/css/_common/components/tags/note.styl 这个 CSS 是全局 CSS，造成污染

### ISSUE THREAD

**Issue by zhangnew** [iissnan/hexo-theme-next#1559]
### Expected behavior (预期行为)
source/css/_common/components/tags/note.styl  这个 CSS 改成针对具体控件

### Actual behavior (实际行为)
source/css/_common/components/tags/note.styl  这个 CSS 写的 note 是 全局的

### Steps to reproduce the behavior (重现步骤)
无。

### NexT Information

Add the  ✔  sign before an item which is affected by this behavior.

NexT Version:

  -  Master
  -  Latest Release
✔ Old version - v5.1.0

NexT Scheme:
  -  All schemes
  -  Muse
✔ Mist
  -  Pisces

### Other Information (Like Browser, System, Screenshots)

scripts/tags/note.js 这个脚本生成了 class="note" ，但是不知道这个是干嘛用的，网页上面没找到这个控件。

https://github.com/iissnan/hexo-theme-next/commit/841c03e690ba3816ecb206b64b6b042c5545fd38 提交信息为：CSS: add bs-callout notes style.

**Comment by ivan-nginx** [comment]
@zhangnew may u write in english? What is it? I don't understand.

**Comment by zhangnew** [comment]
@ivan-nginx `source/css/_common/components/tags/note.styl ` this CSS is a global style, I think it should be wrote for a specific widget. 

I have a `div` with this attribute `class="note"` ，I set a specific style for this `div`, but the style of this `div` is polluted by the  `/note.styl` .

 I want to know what the `/note.styl` wrote for , and why wrote it as a global variable.

**Comment by ivan-nginx** [comment]
@zhangnew > I want to know what the /note.styl wrote for , and why wrote it as a global variable.
`note.styl` is a style for note tag. This mean if u write in .md:
```
{% note %}
Any text
{% endnote %}
```
it will be bs-callout notes style (#1160).
Of course, u may set any custom div in markdown, and if it will be `note`, all styles will be applied for div with this name.

For the second question, i'm not profi about global or not global styles, but there is commented string and i think this is not applied for real code.

Also, if your style conflict with note tag style, u may rename:
1. Your custom div style in all .md files from `note` to `note2` for e.x.
2. From '<div class="note ' to '<div class="note2 ' in `scripts/tags/note.js`.

**Comment by ivan-nginx** [comment]
@zhangnew this is solved your problem?

**Comment by zhangnew** [comment]
Thx. Actually, I think your `note` is wrote for `class="post-body"`( the article content ), just like 
```
.post-body .note {
	padding: ... ;
	margin: ... ;
}
```
Just change all of the `.note` to `.post-body .note` in the `source/css/_common/components/tags/note.styl`
That's why I called `note` is a global style.

My `note` class is wrote for my comments system : isso, but polluted by the global `note` style.

I know I can change my class name, but I think your 'note' write for the `post-body` class is a good idea, then add option in theme config will be better.

**Comment by ivan-nginx** [comment]
@zhangnew ok, fixed in #1697 pull. Also, refactor styles and add some features.

If solved, close issue.

**Comment by zhangnew** [comment]
solved, thank you very much.
